### PR TITLE
Update build system

### PR DESCRIPTION
**Pull request by JonasVautherin**
Multiple things have been changed in this PR, but I believe the most important one is that SwiftGRPC and RxSwift are now installed from Cocoapods in the SDK (nothing changes for the Sample app, though). So it means two things in practice:

* For developing the SDK, one now needs to to install the dependencies using Cocoapods (simply by calling `$ pod install` from the root of the SDK).
* For third-party apps depending on the SDK, the manual installation doesn't work anymore and SwiftGRPC and RxSwift will come automatically from Cocoapods (the backend still needs to be fetch with `$ bash fetch_backend.bash` or compiled separately).

Other changes, FYI:
* The proto files have been updated.
* `dronecore` has been replaced by `dronecode_sdk` in many places.
* Archives are now versioned on Amazon S3. So from now on, it will be possible to depend on an older version of the SDK (starting with `0.1.0`).
* This PR may solve issue #58, but I couldn't try it with my Apple account. I'll try to update that ASAP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,25 +2,13 @@
...
-
-* [CocoaPods](https://cocoapods.org/)
-* Manually
-
-You should use one of these three ways to import the DroneCore-Swift SDK but not multiple. Importing the SDK in multiple ways loads duplicate copies of the SDK into the project and causes compiler errors.
-
 ### CocoaPods
 
-1. The DroneCore-Swift SDK for iOS is available through [CocoaPods](http://cocoapods.org). If you have not installed CocoaPods, install CocoaPods by running the command:
+1. The DroneCore-Swift SDK for iOS is available through [CocoaPods](http://cocoapods.org). CocoaPods can be installed with the following commands:
```
Comment (julianoes) [inline README.md]: Should this also be renamed to DronecodeSDK-Swift?

File: README.md [hunk 2]
```diff
@@ -29,7 +17,7 @@ You should use one of these three ways to import the DroneCore-Swift SDK but not
         use_frameworks!
         
         target :'YourTarget' do
-            pod 'DroneCore-Swift'
+            pod 'DroneCore-Swift', '= 0.1.0'
```
Comment (julianoes) [inline README.md]: I guess not yet?

File: Source/build_dronecode_all.bash [hunk 3]
```diff
@@ -13,7 +13,7 @@ mkdir -p ${BIN_DIR}
 mkdir -p ${DRONECORE_SWIFT_DIR}
 mkdir -p ${DRONECORE_DIR}
 
-# Build dronecore backend and copy it into ${DRONECORE_SWIFT_DIR}/Source/bin where it is needed to build the SDK
+# Build dronecore backend
```
Comment (julianoes) [inline Source/build_dronecode_all.bash]: You could update the URL below.

File: README.md (hunk 1 above)
Comment (JonasVautherin) [reply README.md]: Yes, I'll need to change the podspec and push it to the Cocoapods repo. I'll do that later (ASAP).

File: README.md (hunk 2 above)
Comment (JonasVautherin) [reply README.md]: Exactly xD.

**Pull request merged**